pyanaconda/modules/payloads/source/utils.py

```diff
@@ -84,10 +84,7 @@ def find_and_mount_device(device_spec, mount_point):
     device_path = "/dev/" + matches[0]
 
     try:
-        # FIXME: Add back RO mounting. This was removed because we can't mount one source
-        # RW and RO at the same time. This source is also mounted by IsoChooser dialog in the
-        # SourceSpoke.
-        mount(device_path, mount_point, "auto")
+        mount(device_path, mount_point, "auto", "defaults,ro")
```

Comment:
Could you specify the options as `fstype="auto", options="defaults,ro"`? I was a little confused.

Reply:
I have to set everything as keywords but yes I agree it should be there.